Added destfilename parameter

fgrande opened the pull request:
Hi,

I was wondering how to give an arbitrary name to the compressed filename, and I realized it's always the source directory name.

So I tried a couple of changes:

1) Added DESTFILENAME so it's easy to set the final name for the compressed file
2) changed the way you build the full_dest path, using directly the "self.dest" variable, since it's initially loaded by BaseProcessor

Added the new info to README

Let me know if this is useful and if my changes do sound good to you

Regards
FabioG

somaz94 commented:
Thank you for the PR, but I have some issues.

The includeRoot option is not working properly. You can check the failed workflow to find out.

for example:
- option: tar, false, test2

incorrect workflow
- Executing: tar -cf test2.tar -C /github/workspace/./test2 .

correct workflow
- Executing: tar -cf /github/workspace/./test2/test2.tar -C /github/workspace/./test2 .

I think the idea is good. Should I correct the problem after the pr approval? Or do you want to do it yourself?

fgrande commented:
Hi,
I think I've fixed the dest issue, I removed a part of the logic linked to the include_root switch (not really clear, need to do some test on my pipelines to better understand).

In any case, now Your tests should run fine (not clear if I can somehow run them on my fork)

Let me know if you have further issues with my PR

Thanks
FabioG
